lib/AST/ASTPrinter.cpp

```diff
@@ -3091,13 +3093,9 @@ void PrintAST::visitClassDecl(ClassDecl *decl) {
     printSourceRange(CharSourceRange(Ctx.SourceMgr, decl->getStartLoc(),
                               decl->getBraces().Start.getAdvancedLoc(-1)), Ctx);
   } else {
-    if (!Options.SkipIntroducerKeywords) {
-      if (decl->isExplicitActor()) {
-        Printer.printKeyword("actor", Options, " ");
-      } else {
-        Printer << tok::kw_class << " ";
-      }
-    }
+    if (!Options.SkipIntroducerKeywords)
+      Printer.printIntroducerKeyword(
+          decl->isExplicitActor() ? "actor" : "class");
```

Comment:
Since `actor` is not declared as a keyword, there's no reason to use `tok::kw_*` for introducer printing anymore.